rdr_service/rdr_client/pairing_assigner.py

```diff
@@ -105,6 +105,7 @@ def main(client):
             if client.args.dry_run:
                 logging.info("Dry run, would update participant[%r] to %r.", pairing_key, new_pairing)
             else:
+                print(participant)
```

Comment:
I'm not sure why the pairing_assigner.py is in this PR, but can you remove this print statement?

Reply:
oh i might have added that while dealing with the nph repair i did yesterday, will get that out of there.